guides/common/modules/proc_increasing-the-logging-levels-to-help-with-debugging.adoc

```diff
@@ -116,62 +104,29 @@ You can find the log for {Project} in `/var/log/foreman/production.log`.
 * `/var/log/httpd/foreman_ssl_access.log`
 
 .Procedure
-. Set the logging level in `/etc/foreman/settings.yaml`:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-:logging:
-  :production:
-    :type: file
-    :layout: pattern
-    :level: debug
-----
-. Enable selected loggers in `/etc/foreman/settings.yaml`:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-:loggers:
-  :ldap:
-    :enabled: true
-  :permissions:
-    :enabled: true
-  :sql:
-    :enabled: true
-----
-+
-Note that to see logging from some area, debug logging has to be set.
-. Restart {Project} services:
+. Set the logging level:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-maintain} service restart
+{foreman-installer} --foreman-logging-level debug \
```

Comment:
Should something also be done about
```
 :production:
   :type: file
   :layout: pattern
```
Or is this already included int the file and requires no edits from CLI?

Reply:
> Or is this already included in the file and requires no edits from CLI?

The installer does all the file editing and restarting. That's the goal of this PR.